pages/index.js

```diff
@@ -1,7 +1,59 @@
-export default function HomePage() {
+import { shoppingItems } from "../lib/shoppingItemsData.js";
+import { categories } from "../lib/categoriesData.js";
+
+import Link from "next/link";
+import { useRouter } from "next/router";
```

Comment:
You do not need the `Link `component or the `useRouter` hook to complete this user story. Please delete them. 
```suggestion
```

Reply:
Thanks. I deleted the unnecessary code.